lesson_05/homework_5.2.py

```diff
@@ -0,0 +1,48 @@
+"""Homework 5.2. List of tuples."""
+
+# Given list of tuples (name, surname, age, profession, City location)
+# 1 - Add your new record o the beginning of the given list
+# 2 - In modified list swap elements with indexes 1 and 5 (1<->5). Print result
+# 3 - check that all people in modified list with records indexes 6, 10, 13
+#   have age >=30. Print condition check result
+
+people_records = [
+    ('John', 'Doe', 28, 'Engineer', 'New York'),
+    ('Alice', 'Smith', 35, 'Teacher', 'Los Angeles'),
+    ('Bob', 'Johnson', 45, 'Doctor', 'Chicago'),
+    ('Emily', 'Williams', 30, 'Artist', 'San Francisco'),
+    ('Michael', 'Brown', 22, 'Student', 'Seattle'),
+    ('Sophia', 'Davis', 40, 'Lawyer', 'Boston'),
+    ('David', 'Miller', 33, 'Software Developer', 'Austin'),
+    ('Olivia', 'Wilson', 27, 'Marketing Specialist', 'Denver'),
+    ('Daniel', 'Taylor', 38, 'Architect', 'Portland'),
+    ('Grace', 'Moore', 25, 'Graphic Designer', 'Miami'),
+    ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+    ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+    ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+    ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix'),
+]
+
+# новий запис на початку
+people_records.insert(0, ('Liuda', 'Chunikhina', 37, 'QA', 'Kharkiv'))
+
+# обмін місцями 1 і 5
+temp = people_records[1]
+people_records[1] = people_records[5]
+people_records[5] = temp
+
+# модифікований список
+print('Список зі змінами:')
+for person in people_records:
+    print(person)
+
+# перевірка віку людей з індексами 6, 10 і 13 більше або =30
+age_checked = (
+    people_records[6][2] >= 30 and
+    people_records[10][2] >= 30 and
+    people_records[13][2] >= 30
+)
```

Comment:
Hardcodes better to move as constants to the top of the file.
The best option here to use all() with cycles/ list comprehensions etc.